### PR TITLE
list added

### PR DESCRIPTION
**Pull request by swstica**

### CODE REVIEW

File: lists/lists.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="lists.css">
+    <link rel="stylesheet" href="../components.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Inputs</title>
```
Comment (manish-GH) [inline lists/lists.html]: You probably forgot to rename it to Lists

**Review by manish-GH — COMMENTED** [review]
The code and UI are clean and to the point. 

You could consider adding more styles to your lists. It's like there is no difference between your list and a normal HTML list

It can be merged as of now but do add styles once you get the time.

**Review by sri-rishi — APPROVED** [review]
Great work can't find any issue, feel free to merge

**Review by Anurag021 — COMMENTED** [review]
Code looks good to me for merge.

**Pull request merged**